code/game/machinery/mass_driver.dm

```diff
@@ -180,7 +180,7 @@
 				to_chat(user, "You begin to complete \the [src]...")
 				playsound(get_turf(src), R.usesound, 50, 1)
 				if(do_after(user, 20 * R.toolspeed, target = src) && (R.amount >= 3) && (build == 3))

```

Comment:
This amount check also needs to be adjusted in that case, I think.

Reply:
Good catch